lib/app/features/posts/view/screens/post_detail_view.dart

```diff
@@ -41,32 +42,46 @@ class PostDetailView extends HookConsumerWidget {
         body: AdjustableTextWidget(
           child: Padding(
             padding: EdgeInsets.only(bottom: bottomSheetHeight),
-            child: SingleChildScrollView(
-              child: Column(
-                children: [
-                  if (post.imageUrl != null)
-                    WpaImage(
-                      post.imageUrl!,
-                      fit: BoxFit.fitWidth,
+            child: NotificationListener<ScrollNotification>(
```

Comment:
i thought it would be much easier than this, sorry about that.

Reply:
I just though it looked awkward if the user had scrolled to where the image was offscreen but still had the animation play